test/functionalTest/cases/0970_create_entity/errors_at_creation.test

```diff
@@ -0,0 +1,351 @@
+# 06. POST /v2/entities with boolean Entity::type
+# 07. POST /v2/entities with integer Entity::type
+# 08. POST /v2/entities without Attribute::name
+# 09. POST /v2/entities with integer Attribute::name
+# 10. POST /v2/entities with boolean Attribute::name
+# 11. POST /v2/entities with empty Attribute::value
+#
+
+echo "00. POST /v2/entities with Entity::id AND Entity::idPattern"
+echo "==========================================================="
+payload='{
+  "id": "id",
+  "idPattern": "pattern",
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "01. POST /v2/entities without Entity::id"
+echo "========================================"
+payload='{
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "02. POST /v2/entities with empty Entity::id"
+echo "==========================================="
+payload='{
+  "id": "",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "03. POST /v2/entities with empty Entity::idPattern"
+echo "=================================================="
+payload='{
+  "idPattern": "",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "04. POST /v2/entities with boolean Entity::id"
+echo "============================================="
+payload='{
+  "id": true,
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "05. POST /v2/entities with integer Entity::id"
+echo "============================================="
+payload='{
+  "id": 12,
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "06. POST /v2/entities with boolean Entity::type"
+echo "==============================================="
+payload='{
+  "id": "ID",
+  "type": false,
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "07. POST /v2/entities with integer Entity::type"
+echo "==============================================="
+payload='{
+  "id": "ID",
+  "type": 19,
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "08. POST /v2/entities without Attribute::name"
+echo "============================================="
+payload='{
+  "id": "ID",
+  "type": "WT",
+  "": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "09. POST /v2/entities with integer Attribute::name"
+echo "=================================================="
+payload='{
+  "id": "ID",
+  "type": "WT",
+  6: "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "10. POST /v2/entities with boolean Attribute::name"
+echo "=================================================="
+payload='{
+  "id": "ID",
+  "type": "WT",
+  true: "1"
```

Comment:
It does, as you will see later on.
Not really a problem, for a functest.

NTC